callisto/delivery/hashers.py

```diff
@@ -0,0 +1,231 @@
+import base64
```

Comment:
they're more or less the same, except made less dynamic (there's no `_load_library()` functions for instance) and with a few other things taken out. also, the `split_encoded()` methods were added. I figured it would be best to change the names though in case of any possible weird conflict issues and there were enough differences to make just importing or subclassing django's methods tricky from what I could tell